src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.25.1"
+__version__ = "0.25.2"
```

Comment:
@harini-venkataraman please assess if the changes add are breaking or not. Also FYI `0.25.1` itself has not been published yet.
Usually for minor version increments in adapters also we increment the minor version of SDK

Reply:
@chandrasekharan-zipstack Got it. Ignored the patch version and bumped the minor version itself.